### ISSUE TITLE
[LanguageServer] Hover is not working for local and non local cross ref.

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-core#20]
the tooltip that is calculated is filtered out inside hoverservice

`val textRegion = containedElement.significantTextRegion`
usually never comes true for cross refs that are no contained is wrapper object

@akosyakov what do you think?

**Comment by cdietrich** [comment]
Problem can be reproduced as follows
change test grammar to
```TypeDeclaration:
	'type' name=ID ('extends' extends=[TypeDeclaration])? '{'
	   properties+=Property*
	'}';
```
use this test
```
	@Test
	def void testHover_04() {
		val referenceModel = '''
			/**
			 * Some documentation.
			 */	
			type Foo {
			}
		'''
		val fileUri = 'MyModel2.' + fileExtension -> referenceModel
		initialize
		open(fileUri, referenceModel)
		testHover[
			model = '''
				type Bar extends Foo {
				}
			'''
			line = 0
			column = 'type Bar extends F'.length
			expectedHover = '''
				[[0, 17] .. [0, 20]]
				Some documentation.
			'''
		]
	}
```

**Comment by cdietrich** [comment]
resolved with bf9021d82280876a5064919a69de347bf30371db
